# Path2DWithParams.msg

Pose2DWithParams[] path
float64 distance_threshold
float64 angle_threshold